src/Form/Type/Procuration/ElectionContextType.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace AppBundle\Form\Type\Procuration;
+
+use AppBundle\Entity\Election;
+use AppBundle\Procuration\ElectionContext;
+use AppBundle\Repository\ElectionRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ElectionContextType extends AbstractType
+{
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver
+            ->setDefaults([
+                'data_class' => ElectionContext::class,
+            ])
+        ;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('elections', EntityType::class, [
+                'multiple' => true,
+                'expanded' => true,
+                'class' => Election::class,
+                'query_builder' => function (ElectionRepository $repository) {
```

Comment:
where/how did you get the ElectionRepository here ?

Reply:
Thanks to the entity type and its `query_builder` option.

Reply:
@salahm FYI https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Doctrine/Form/Type/EntityType.php#L32.

So you can either type hint the generic `EntityRepository` class or a custom one if it exists.